.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ orbs:
 executors:
   executor_med:  # 2cpu, 4G ram
     docker:
-      - image: circleci/openjdk:11.0.8-jdk-buster
+      - image: cimg/openjdk:11.0.3
```

Comment:
Would the latest cimg/openjdk image also work? Looks like there is one with a 11.0.13 tag available

Reply:
Yep, should do. My bad - I'll fix that up

Reply:
done